Makefile

```diff
@@ -531,14 +533,24 @@ else
 endif
 
 docker-push-kat-client: kat-client-docker-image
+ifeq ($(DOCKER_REGISTRY),-)
+	@echo "No DOCKER_REGISTRY set"
+else
 	@echo 'PUSH $(KAT_CLIENT_DOCKER_IMAGE)'
 	@set -o pipefail; \
 		docker push $(KAT_CLIENT_DOCKER_IMAGE) | python releng/linify.py push.log
+endif
 
 docker-push-kat-server: kat-server-docker-image
+ifeq ($(DOCKER_REGISTRY),-)
+	@echo "No DOCKER_REGISTRY set"
+else
 	@echo 'PUSH $(KAT_SERVER_DOCKER_IMAGE)'
 	@set -o pipefail; \
 		docker push $(KAT_SERVER_DOCKER_IMAGE) | python releng/linify.py push.log
+endif
+
+docker-push-kat: docker-push-kat-client docker-push-kat-server
```

Comment:
What do you mean by that?
Btw, I've added this target because I think you were right that push on every test run will be slow. Instead, I now run `make docker-push-kat ambassador-docker-image test`

Reply:
Geez. Need more caffeine when reviewing, I think. 😛 I read that line and was thinking that I was looking at the `test` target.

Why do you need to push the `kat` images, but not the Ambassador image?

Reply:
Sure, I could do `make docker-push-kat docker-push test` as well. Just a creature of habit here ;)